spec/ruumba/iterators_spec.rb

```diff
@@ -0,0 +1,63 @@
+# @author Eric Weinstein <eric.q.weinstein@gmail.com>
```

Comment:
@aclemons You can include yourself as the author here if you like

Reply:
Ok, added now (force pushed).